tmt/log.py

```diff
@@ -797,13 +796,13 @@ def debug(
                 message_topic=topic)
             )
 
-    def warn(
+    def warning(
```

Comment:
Just to be safe, can you create a `def warn` and add `DeprecationWarning` in it? I think I was using it at some point when writing external plugins.

Reply:
Thanks. Added in https://github.com/teemtee/tmt/pull/2842/commits/4524b65129be20c3dd42ac48b9333fe0e8889ef9